testool/src/main.rs

```diff
@@ -21,7 +17,7 @@ use strum::EnumString;
 const REPORT_FOLDER: &str = "report";
 const CODEHASH_FILE: &str = "./codehash.txt";
 
-#[macro_use]
+// #[macro_use]
```

Comment:
This change is required?

Reply:
A warning of unused import, oh a better way is to remove the whole use sentence, done.